accounts/jobs/daily/collect_payouts.py

```diff
@@ -28,7 +33,43 @@ def execute(self):
             Payout.objects.get_or_create(
                 user_profile=user,
                 date=today,
-                amount=amount,
+                defaults={
+                    "amount": amount,
+                },
             )
 
+        # Get all referral which do not requested or paid
+        referral_users = UserProfileReferralHit.objects.filter(
+            payment_status=UserProfileReferralHit.NONE,
+        ).order_by("pk")
+        number_of_referral = UserProfileReferralHit.objects.filter(
+            ~Q(payment_status=UserProfileReferralHit.NONE)
+        ).count()
+        total = 100
+        # Create referral payouts
+        for referral in referral_users:
+            activate_days = referral.referral_hit.hit_user.payouts.count()
+            if activate_days == 90:
+                number_of_referral += 1
+
+                amount = total * number_of_referral / (
```

Comment:
@autn let's make this amount calculation a helper function that includes a docstring with a link to my SO question, MyPy type hints, and a unit test.

https://math.stackexchange.com/questions/4290947/how-can-i-craft-a-function-for-my-referral-program-with-a-horizontal-asymptote